homeassistant/components/unifi/unifi_entity_base.py

```diff
@@ -75,7 +75,7 @@ def async_update_callback(self) -> None:
 
     async def options_updated(self) -> None:
         """Config entry options are updated, remove entity if option is disabled."""
-        raise NotImplementedError
+        pass  # pylint: disable=unnecessary-pass
```

Comment:
This shouldn't be changed at the moment. Rather add an empty method in the subclass.